conans/client/tools/system_pm.py

```diff
@@ -13,6 +13,8 @@ def __init__(self, runner=None, os_info=None, tool=None):
         self._sudo = (env_sudo != "False" and env_sudo != "0")
         if env_sudo is None and os.name == 'posix' and os.geteuid() == 0:
             self._sudo = False
+        if env_sudo is None and os.name == 'nt':
```

Comment:
sudo doesn't make sense in Windows at all? 
Then maybe it can make ``self._sudo = False`` irrespective of ``env_sudo``

Reply:
Any feedback on this? Many thanks!